internal/k6runner/k6runner.go

```diff
@@ -307,8 +313,18 @@ NEXT_RECORD:
 type HttpRunner struct {
 	url    string
 	logger *zerolog.Logger
+	// backoff sets the minimum amount of time to wait before retrying a request. nth attempt waits n times this value,
+	// plus some jitter.
+	backoff time.Duration
+	// graceTime tells the HttpRunner how much time to add do the script timeout to form the request timeout.
+	graceTime time.Duration
 }
 
+const (
+	defaultBackoff   = 10 * time.Second
+	defaultGraceTime = 20 * time.Second
```

Comment:
Grace time is another way to say "the maximum amount of lag we can tolerate in the queues". https://github.com/grafana/sm-k6-runner/pull/214 defines a grace time in the proxy side of 1 minute, so making this smaller effectively makes the grace time of the system smaller. However, defaulting to 1m of grace time means there won't be any retries for checks with frequency <= 1m.